--- vde-1.5.11/vdetaplib/libvdetap.c.orig	2005-03-24 15:31:16.000000000 +0100
+++ vde-1.5.11/vdetaplib/libvdetap.c	2012-10-25 18:46:14.461514817 +0200
@@ -13,6 +13,7 @@
 #define __USE_LARGEFILE64
 #include <fcntl.h>
 #include <errno.h>
+#include <limits.h>
 #include <signal.h>
 #include <linux/ioctl.h>
 #include <linux/if.h>
--- vde-1.5.11/vde_switch.c.orig	2005-05-01 15:35:34.000000000 +0200
+++ vde-1.5.11/vde_switch.c	2012-10-25 18:46:43.134846950 +0200
@@ -22,6 +22,7 @@
 #include <syslog.h>
 #include <libgen.h>
 #include <endian.h>
+#include <limits.h>
 #include <vde.h>
 #include <switch.h>
 #include <port.h>
